src/Microsoft.AspNet.Mvc.Razor/RazorView.cs

```diff
@@ -233,13 +238,18 @@ private Task RenderPageCoreAsync(IRazorPage page, ViewContext context)
             }
         }
 
-        private IRazorPage GetLayoutPage(ViewContext context, string layoutPath)
+        private IRazorPage GetLayoutPage(ViewContext context, string executingFilePath, string layoutPath)
         {
-            var layoutPageResult = _viewEngine.FindPage(context, layoutPath);
+            var layoutPageResult = _viewEngine.GetPage(executingFilePath, layoutPath, isPartial: true);
```

Comment:
Not sure I understand. Is the parameter name just bad? What behavior are we trying to get (or avoid) by passing in this seemingly-wrong parameter?